src/modules/openrccfg/openrccfg.conf

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Most modules have a short bit of documentation about the configuration in general, above this --- comment line.